src/EbmlString.cpp

```diff
@@ -58,12 +58,10 @@ filepos_t EbmlString::ReadData(IOCallback & input, ScopeMode ReadFully)
   if (ReadFully == SCOPE_NO_DATA)
     return GetSize();
 
-  if (GetSize() == 0) {
-    Value.clear();
+  Value.clear();
```

Comment:
If Value has a size of 10 and we call resize(8) it will likely no do any allocation. Here you are forcing a reallocation in *all* cases. IMO it's not good.

Reply:
Avoids memset